### PR TITLE
Add e2e tests for OLTP metrics and fix a bug in oc->otlp translation

### PR DESCRIPTION
**Pull request by dmitryax**
- Add e2e tests for OLTP metrics
- Fix a bug in OC->OTLP translation when metrics with no labels were failing

### CODE REVIEW

File: testbed/go.sum [hunk 1]
```diff
@@ -457,6 +457,7 @@ github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742/go.mod h1:bx2lN
 github.com/modern-go/reflect2 v1.0.1 h1:9f412s+6RmYXLWZSEzVVgPGK7C2PphHj5RJrvfx9AWI=
 github.com/modern-go/reflect2 v1.0.1/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
 github.com/mozilla/tls-observatory v0.0.0-20190404164649-a3c1b6cfecfd/go.mod h1:SrKMQvPiws7F7iqYp8/TX+IhxCYhzr6N/1yb8cwHsGk=
+github.com/mozilla/tls-observatory v0.0.0-20200220173314-aae45faa4006/go.mod h1:SrKMQvPiws7F7iqYp8/TX+IhxCYhzr6N/1yb8cwHsGk=
```
Comment (tigrannajaryan) [inline testbed/go.sum]: Please `go mod tidy` in testbed to make sure we don't have unneeded dependency references.

**Comment by bogdandrutu** [comment]
@dmitryax did you run `go mod tidy`?

**Comment by dmitryax** [comment]
> @dmitryax did you run `go mod tidy`?

yes, I did

**Comment by dmitryax** [comment]
@bogdandrutu bad auto-rebase. let me submit a fix

**Pull request merged**